packages/apps/sales/src/i18n/en.json

```diff
@@ -1,8 +1,10 @@
 {
   "Sales_Sales": "Sales",
+  "Sales_ProductService": "Products & services",
```

Comment:
In AOS it's "Products & services" but in the ticket it's "Articles & services", so I don't know